src/controllers/playback/video/index.js

```diff
@@ -1125,7 +1147,7 @@ import { appRouter } from '../../../components/appRouter';
                     // Ignores gamepad events that are always triggered, even when not focused.
                     if (document.hasFocus()) { /* eslint-disable-line compat/compat */
                         playbackManager.fastForward(currentPlayer);
-                        showOsd();
+                        showOsd(btnFastForward);
                     }
                     break;
```

Comment:
Hmm, we can probably remove the gamepad bindings as they are mapped to the arrow keys.